wiki/en/Installation-for-iOS

```diff
@@ -0,0 +1,41 @@
+---
+layout: wiki
+title: "Installation for iOS"
+lang: "en"
+permalink: "/wiki/Installation-for-iOS"
+---
+
+{% include breadcrumb.html root="Using Jamulus" branch1="Getting Started" branch1-url="Getting-Started" %}
+
+
+# Installation for iOS
+
+Make sure you've already read the [Getting Started](Getting-Started) page.
+
+## Things to note about iOS
+Although Jamulus on iOS works quite well on new devices if everything is set up correctly, we strongly recommend only doing so if you have an audio interface and an USB-Ethernet adapter. To connect both devices to your iOS device, you should use something like the [Lightning to USB Camera Adapter](https://www.apple.com/uk/shop/product/MD821ZM/A/lightning-to-usb-camera-adapter) and a (powered USB-hub).
+If you have don't own a PC, we suggest you to buy a [Raspberry Pi](https://www.raspberrypi.org/){: target="_blank" rel="noopener noreferrer" } which is an inexpensive and small device that performs very well with Jamulus. iOS support is just a proof of concept and needs certain skills.
+
+## Install the iOS PoC (for experienced users only)
+
+**Attention:** in order to install Jamulus on your device, you must run macOS. Also - if you don't pay for a apple developer account, Jamulus will only run for at most 7 days on your device.
+If you do want to try Jamulus on iOS:
```

Comment:
Not sure why is this here and not in https://github.com/jamulussoftware/jamulus/blob/master/COMPILING.md#ios ? Seems oddly split into two (and I don't understand it - how do you actually get it onto your iOS device? But then I'm not familiar with iOS apps,

Reply:
See my other PR. Maybe that’s enough? We might be able to remove the Xcode step then.